tests/rpc-maas-overrides.yml

```diff
@@ -31,10 +31,12 @@ bridges:
 
 
 # Set the vip address
-external_lb_vip_address: 127.0.0.1
-internal_lb_vip_address: 127.0.0.1
+external_lb_vip_address: 10.1.1.51
+internal_lb_vip_address: 10.2.1.51
 internal_vip_address: "{{ internal_lb_vip_address }}"
 
+# Define the galera monitoring user
```

Comment:
No, the "galera_server" role creates the user and in order for our "haproxy_server" role to allow traffic to the galera endpoint we have to have the monitoring user defined and provided into the haproxy config. I could just hard code it, but I put it here for greater transparency. Maybe I should?